frontend/src/cards/ui/Sources.tsx

```diff
@@ -72,6 +85,9 @@ export function Sources(props: {
 
   let datasetIds = getDatasetIdsFromResponses(props.queryResponses);
```

Comment:
might consider `unstrippedDatasetIds` or something... in general i like avoiding mutating the variables just because it's easier to see what should be there at each step by the variable name alone